src/protected-api-access-protected-api/README.md

```diff
@@ -21,40 +24,19 @@ urlFragment: ms-identity-docs-code-csharp
 This ASP.NET Core minimal web API uses the Microsoft identity platform to protect an endpoint (require authorized access), and also accesses Microsoft Graph on behalf of the user. The API uses [ASP.NET Core Identity](https://docs.microsoft.com/aspnet/core/security/authentication/identity?view=aspnetcore-6.0) interacting with the [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/azure/active-directory/develop/msal-overview) to protect its endpoint.
 
 ```console
-$ curl https://localhost:5001/me -H "Authorization: Bearer {valid-access-token}"
+$ curl https://localhost:5001/api/me -H "Authorization: Bearer {valid-access-token}"
 {
-   "version": "1.1",
-   "content": {
-      "headers": [
-         {
-            "key": "Content-Type",
-            "value": [
-               "application/json; odata.metadata=minimal; odata.streaming=true; IEEE754Compatible=false; charset=utf-8"
-            ]
-         }
-      ]
-   },
-   "statusCode": 200,
-   "reasonPhrase": "OK",
-   "headers": [
-     ...
-   ],
-   "trailingHeaders": [],
-   "requestMessage": {
-      "version": "1.1",
-      "versionPolicy": 0,
-      "content": null,
-      "method": {
-         "method": "GET"
-      },
-      "requestUri": "https://graph.microsoft.com/v1.0/me/",
-      "headers": [
-        ...
-      ],
-      "properties": {},
-      "options": {}
-   },
-   "isSuccessStatusCode": true
+  "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity",
+  "businessPhones": [],
+  "displayName": "Maui",
+  "givenName": null,
+  "jobTitle": "dev",
+  "mail": null,
+  "mobilePhone": null,
+  "officeLocation": null,
+  "preferredLanguage": null,
+  "surname": null,
+  "id": "cff40dac-17ea-4183-9caf-65f2ee90c562"
```

Comment:
<3  Thanks.  I saw this last week and was going to mention it and then quickly forgot about it.